cobra/cmd/helpers.go

```diff
@@ -167,13 +185,20 @@ func guessProjectPath() {
 		// hardest case.. just a word.
 		if projectBase == "" {
 			x, err := getWd()
-			if err == nil {
-				projectPath = filepath.Join(x, inputPath)
-				return
+			if err != nil {
+				er(err)
 			}
-			er(err)
+			projectPath = filepath.Join(x, inputPath)
+			return
 		} else {
-			projectPath = filepath.Join(srcPath, projectBase, inputPath)
+			for _, srcPath := range srcPaths {
+				fpath := filepath.Join(srcPath, projectBase, inputPath)
+				if b, _ := exists(fpath); b {
+					projectPath = fpath
+					return
+				}
+			}
+			projectPath = filepath.Join(srcPaths[0], projectBase, inputPath)
```

Comment:
A new function that wraps these three variables `srcPath`,`inputPath` and one of `""`, `"github.com"`and `projectBase`. Any comments?